pkg/grizzly/parsing.go

```diff
@@ -98,14 +98,15 @@ var script string
 // ParseJsonnet evaluates a jsonnet file and parses it into an object tree
 func ParseJsonnet(jsonnetFile string, opts Opts) (Resources, error) {
 
-	script := fmt.Sprintf(script, jsonnetFile)
-	vm := jsonnet.MakeVM()
-	absoluteFile, err := filepath.Abs(jsonnetFile)
+	currentWorkingDirectory, err := os.Getwd()
 	if err != nil {
-		return nil, err
+		log.Fatal(err)
```

Comment:
Grizzly doesn't use `log.Fatal`, can you pass back a sensible error message?

Reply:
Thanks, good catch. I think the command can only fail in 'non-sensible' situations :wink:, like the current working directory being deleted.